lib/livebook_cli/server.ex

```diff
@@ -127,6 +127,12 @@ defmodule LivebookCLI.Server do
       |> Livebook.Utils.browser_open()
     end
   end
+  
+  defp open_from_options(base_url, opts, extra_args)
+    path = Path.expand(Enum.fetch!(extra_args, 0))
```

Comment:
```suggestion
  defp open_from_options(base_url, opts, [path])
    path = Path.expand(path)
```

And we need to raise if more than one argument is given. :)

Reply:
ah yes, was wondering we need a catch all clause